.travis.yml

```diff
@@ -26,6 +26,8 @@ before_script:
   - java -jar ./target/cargo-daemon-webapp-${CARGO_DAEMON_WEBAPP_VERSION}.war &
 
 script: 
+  - mvn xml:check-format
+  - mvn formatter:validate
```

Comment:
- [ ] Please see [pom.xml](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/issues/488_apply_validation_of_code_format/pom.xml#L289-L293). These command act "env", "domain", "web" projects only.
It's necessary to consider to check "initdb" and "selenium" project.